src/lake/Lake/Util/Newline.lean

```diff
@@ -0,0 +1,10 @@
+/-
+Copyright (c) 2023 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro
+-/
+
+namespace Lake
+
+/-- This is the same as `String.replace text "\r\n" "\n"`, but more efficient. -/
+@[inline, deprecated] partial def crlf2lf (text : String) : String := text.crlfToLf
```

Comment:
FYI, you could instead remove the `Lake.Util.Newline` import from `src/lake/tests/toml/Test.lean`.